image-builder/README.md

```diff
@@ -14,12 +14,97 @@ We use the Docker's building system and convert it at the end to a disk image th
 [More info](https://github.com/scaleway/image-tools)
 
 
-:warning: **Documentation incoming** :warning:
+## How to build a custom image (using [scw](https://github.com/scaleway/scaleway-cli))
+
+**My custom image's description**
+- based on [ubuntu vivid](https://github.com/scaleway/image-ubuntu)
+- `cowsay` pre-installed
+
+---
+
+##### 1 Making the environment
+
+```bash
+root@yourmachine> scw run --rm --name="buildcowsay" builder
+...SKIP
+*****************************************************************************
+
+   Welcome on image-builder.
+   Here, you will be able to do your own images.
+
+   To configure your environment please run:
+
+   $> image-builder-configure
+
+*****************************************************************************
+...SKIP
+root@buildcowsay> image-builder-configure
+Login (cloud.scaleway.com):                     # yourmail
+Password:                                       # yourpassword
+Do you want to upload an SSH key ?
+[0] I don t want to upload a key !
+[1] id_rsa.pub
+Which [id]: 1                                   # you must upload your local key
+S3_URL (s3://new_image): s3://vividcowsay
+
+###
+### Warning
+### Please run: source /etc/bash.bashrc
+###
+
+root@buildcowsay> source /etc/bash.bashrc
+root@buildcowsay> mkdir vivid-cowsay
+root@buildcowsay> cp Makefile vivid-cowsay
+root@buildcowsay> cd vivid-cowsay
+root@buildcowsay> touch Dockerfile
+root@buildcowsay> ls -l
+total 4
+-rw-r--r-- 1 root root   0 Aug 28 15:35 Dockerfile
+-rw-r--r-- 1 root root 146 Aug 28 15:19 Makefile
+```
+
+##### 2 Generating a Dockerfile
+
+**Copy-Paste** this in your `Dockerfile` [see more](https://docs.docker.com/reference/builder/)
+```dockerfile
+# base image - ubuntu:vivid
+FROM armbuild/scw-distrib-ubuntu:vivid
+
+# install cowsay
+RUN apt-get install -y cowsay
+```
```

Comment:
Add a link to a list of examples: https://github.com/QuentinPerez/image-tools/tree/doc#official-images-built-with-image-tools